clustering/ee/cache/src/main/java/org/wildfly/clustering/ee/cache/scheduler/PortableConcurrentDirectDeque.java

```diff
@@ -820,7 +812,7 @@ public PortableConcurrentDirectDeque(Collection<? extends E> c) {
         // Copy c into a private chain of Nodes
         Node<E> h = null, t = null;
         for (E e : c) {
-            checkNotNull(e);
+            checkNotNullParamWithNullPointerException("e", e);
```

Comment:
"e" is not a parameter of this method. Is there an Assert.checkNotNullArrayParam(...) equivalent for collections?